src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSNodeFile.cs

```diff
@@ -53,11 +53,11 @@ internal PSNodeFile(Microsoft.Azure.Batch.NodeFile omObject)
             }
         }
         
-        public string Name
+        public string Path
```

Comment:
same here.

Reply:
My intent is to just leave this as is (no obsolete) since this is a breaking change, unless otherwise suggested by one of the powershell maintainers.

Reply:
This needs to be specifically documented (with an example) in the upcoming-breaking changes

Reply:
Done